resize_data_part_launcher.target-sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+# Copy the resize script to tmpfs and run it.
+# We have to do this so that we can unmount the partition.
+echo 'Launching data partition resizer...'
+cp /media/data/resize_data_part.sh /tmp/resize_data_part.sh
+chmod +x /tmp/resize_data_part.sh
+exec /tmp/resize_data_part.sh
```

Comment:
You can do all of this in a single script, no need to add another one.
Make it check that $0 is /tmp/resize_data_part.sh, and if not, copy $0 to that location then exec.

Reply:
Done